### PR TITLE
executor: set column data to zero value when calling `ResizeXXX`

### PR DESCRIPTION
**Pull request by qw4990**
### What problem does this PR solve? 
Set `Column` data to zero value when calling `ResizeXXX` and the `isTrue` argument is false for safety.

### Check List 

Tests 

 - Unit test

### CODE REVIEW

File: util/chunk/column.go [hunk 1]
```diff
@@ -251,6 +251,11 @@ func (c *Column) resize(n, typeSize int, isNull bool) {
 	} else {
 		c.data = make([]byte, sizeData)
 	}
+	if !isNull {
+		for i := 0; i < sizeData; i++ {
+			c.data[i] = 0
+		}
```
[the remark this answers is not in the document]
Comment (qw4990) [reply util/chunk/column.go]: updated.

File: util/chunk/column.go [hunk 2]
```diff
@@ -251,6 +255,11 @@ func (c *Column) resize(n, typeSize int, isNull bool) {
 	} else {
 		c.data = make([]byte, sizeData)
 	}
+	if !isNull {
+		for j := 0; j < sizeData; j += len(emptyBuf) {
+			copy(c.data[j:], emptyBuf)
```
Comment (XuHuaiyu) [inline util/chunk/column.go]: Will `c.data[last_j:sizeData]` not filled if `sizeData % len(emptyBuf)` != 0

Comment (zz-jason) [reply util/chunk/column.go]: No, it won't.

**Pull request merged**